roles/contiv/tasks/cleanup.yml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
@srampal playbooks really shouldn't be in tasks directory. Maybe move this to playbooks/adhoc